### PR TITLE
Close 4542 -- Update styles for RocketCDN banner

### LINKED ISSUE #4542: Update Styles for RocketCDN
Following the RocketCDN design, the frame for RocketCDN inside the WP Rocket plugin has been re-designed too.
You'll find attached the new design (with/without promo).

The steps coming after clicking the button remain the same.

RocketCDN no promo
RocketCDN promo

### PR DESCRIPTION
**Pull request by iCaspar**
## Description

This is to change the layout/styling of the RocketCDN banner to conform to the new RocketCDN branding.

Fixes #4542

## Type of change
- [X] Enhancement (non-breaking change which improves an existing functionality)

## Is the solution different from the one proposed during the grooming?
Same as grooming.

## How Has This Been Tested?
Observed new layout/styles on development environment.

# Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by remyperona — APPROVED** [review]

**Comment by iCaspar** [comment]
@wp-media/qa see the design comps in the related issue for UI criteria.

**Comment by Mai-Saad** [comment]
@iCaspar @engahmeds3ed Almost all tests pass except for the info icon (still not matching design)
Screenshot from 2021-12-07 12-27-12
[testrail-report-324.pdf](https://github.com/wp-media/wp-rocket/files/7667663/testrail-report-324.pdf)

**Comment by mostafa-hisham** [comment]
I had a call with @Mai-Saad 
please change the style to this to center the "i" and add padding

```
.wpr-rocketcdn-cta-footer a:before {
    content: "i";
    border: 1px solid;
    margin-right: 8px;
    width: 14px;
    height: 12px;
    display: inline-block;
    border-radius: 14px;
    text-align: center;
    vertical-align: text-bottom;
    font-size: 0.75rem;
    line-height: 1;
    line-height: 110%;
    font-weight: 900;
    padding-top: 1px;
    padding-bottom: 1px;
    position: relative;
    top: 1px;
    }
```

**Comment by engahmeds3ed** [comment]
Pushed a small change now to load it as svg image :D

@Mai-Saad ready for review now

File: inc/Engine/CDN/RocketCDN/views/cta-big.php [hunk 1]
```diff
@@ -63,33 +78,17 @@
...
-						printf(
-							// translators: %s = price of RocketCDN subscription.
-							esc_html__( '%s / month', 'rocket' ),
-							'<span class="wpr-title1">$' . esc_html( $data['current_price'] ) . '</span>'
-						);
-						?>
+						<span class="wpr-rocketcdn-cta-currency-minor">$</span>
+						<span class="wpr-rocketcdn-cta-currency-major"><?php esc_html_e( substr( $data['current_price'], 0, strpos( $data['current_price'], '.' ) ) ); ?></span>
+						<span class="wpr-rocketcdn-cta-currency-minor"><?php esc_html_e( substr( $data['current_price'], strpos( $data['current_price'], '.' ) ) ); ?>
+						<?php if ( ! empty( $data['regular_price'] ) ): ?>*<?php endif; ?>
```
Comment (engahmeds3ed) [inline inc/Engine/CDN/RocketCDN/views/cta-big.php]: What is the use of this line, this is adding two `*` when discount is there, or what am I missing here?
image

**Review by Mai-Saad — APPROVED** [review]
Working as expected

**Pull request merged**